docker/.env.dist

```diff
@@ -0,0 +1,5 @@
+# This file is a template for defining the environment variables
+# Set the application config values here
+
```

Comment:
Could we also mention that these are optional, and not required to be specified?